io-page-unix.opam

```diff
@@ -7,9 +7,8 @@ homepage: "https://github.com/mirage/io-page"
 bug-reports: "https://github.com/mirage/io-page/issues"
 depends: [
   "ocaml" {>= "4.02.3"}
-  "io-page"
-  "dune" {build & >= "1.0"}
-  "configurator" {build}
+  "dune"
```

Comment:
should still be a build dependency, no? (same below)

Reply:
its not safe to use `{build}` with dune (or many other packages) -- see https://github.com/ocaml/opam/issues/3850

Reply:
if you think this is a problem, I'd suggest to actually solve it for all the packages (and document why more publically, not in 2-3 github issues and PRs), instead of "fixing" a single package, leaving the (mirage-opam-)universe in a pretty inconsistent state (to me, it occured https://github.com/mirage/arp/pull/15 that having dune as a full dependency may be an issue for mirage-dune!? //cc @TheLortex)

Reply:
It's being addressed separately here; https://github.com/ocaml/opam-repository/issues/14185

Fixing it when we come across it is fine for now until the bigger opam-repo PR happens. I doubt very much it'll affect mirage-dune